### ISSUE TITLE
editor-styles not loaded on production.

### ISSUE THREAD

**Issue by bdfpplinsinga** [cearls/timberland#45]
Hi Chris, 

Im having a issue with the backend styles on my production site. I know i need to run the build command and i do so. Im having a staging and a production site. Both are on the same server. Staging is just a subdomain of the production version. The weird thing is that on my staging site all works as expected, but on my production site it doestn seem to load the editor styles.

Any idea? I tried a lot but no succes 

Maybe it has anything to do with the following: My subdomains are:
/var/www/vhosts/domain.nl/test.domain.nl/wp-content/themes/timberland

My main has the httpdocs:
/var/www/vhosts/domain.nl/httpdocs/wp-content/themes/timberland

**Comment by cearls** [comment]
Hi, can you confirm that all the expected files are in the assets build directory? Also, ensure that the file names in `theme/assets/dist/.vite/manifest.json` match the asset filenames.

<img width="281" alt="Screenshot 2024-08-15 at 10 39 26 AM" src="https://github.com/user-attachments/assets/5f36dec3-589d-4301-846d-0972dc607ac8">

**Comment by bdfpplinsinga** [comment]
Yes they ar all there. It look like the add_editor_styles function just doesn't run/work.

**Comment by cearls** [comment]
What versions of WordPress are your staging and production sites running? I'm working on a new project using the latest version of WordPress and noticed that block styles in the editor are being overridden.

**Comment by ineptian** [comment]
@bdfpplinsinga What was it that fixed this for you? I just ran into this for the first time. Never had an issue before not sure what is different this time around.
